controls/SV-230466.rb

```diff
@@ -12,7 +12,7 @@
 generated. This set of events is typically a subset of the list of all events
 for which the system is capable of generating audit records.
 
-    DoD has defined the list of events for which RHEL 8 will provide an audit
+  #{input('org_name')[:acronym]} has defined the list of events for which RHEL 8 will provide an audit
```

Comment:
This control has check text that needs to be parameterized for unsuccessful logon attempts and fail interval.

Reply:
fixed: https://github.com/ssayed118/redhat-enterprise-linux-8-stig-baseline/commit/639f9115af83a3f73e368c486fe8af1d20e7a912